types/mui-datatables/index.d.ts

```diff
@@ -41,11 +42,14 @@ export interface MUIDataTableState {
     filterData: any[];
     filterList: string[][];
     page: number;
+    previousSelectedRow: null | { index: number; dataIndex: number };
     rowsPerPage: number;
     rowsPerPageOptions: number[];
     searchText: string | null;
+    searchProps: null | React.HTMLAttributes<HTMLInputElement>;
```

Comment:
```suggestion
    searchProps: React.HTMLAttributes<HTMLInputElement> | null;
```

Nit, but it is a convention to put `null` after the actual type.